pkg/agent/nodeportlocal/npl_agent_test.go

```diff
@@ -389,24 +412,30 @@ func TestPodDelete(t *testing.T) {
 }
 
 // TestPodAddMultiPort creates a Pod with multiple ports and verifies that the Pod's NPL annotation
-// and the local port table are updated correctly.
+// and the local port table are updated with only one port used as target port of the Service.
 func TestPodAddMultiPort(t *testing.T) {
 	testSvc := getTestSvc()
 	testPod := getTestPod()
-	newPort := 90
+	newPort1 := 91
+	newPort2 := 92
 	testPod.Spec.Containers[0].Ports = append(
 		testPod.Spec.Containers[0].Ports,
-		corev1.ContainerPort{ContainerPort: int32(newPort)},
+		corev1.ContainerPort{ContainerPort: int32(newPort1)},
+	)
+	testPod.Spec.Containers[0].Ports = append(
+		testPod.Spec.Containers[0].Ports,
+		corev1.ContainerPort{ContainerPort: int32(newPort2)},
```

Comment:
I'm still trying to understand why we need to add 2 new ports here (91 and 91). Neither of these is a target port, so neither of these is used for NPL. We only use the default port (90), yet the test is called multi-port. It feels like 90 and 91 should both be NPL ports, but not 92.